### PR TITLE
feat: add account_sid to tts hash key and confiurable TTL for cached file

### PR DESCRIPTION
**Pull request by xquanluu**
#9 #10

### CODE REVIEW

File: lib/synth-audio.js [hunk 1]
```diff
@@ -95,8 +95,9 @@ async function synthAudio(client, logger, stats, {
   } else  if (vendor.startsWith('custom')) {
     assert.ok(credentials.custom_tts_url, `synthAudio requires custom_tts_url in credentials when ${vendor} is used`);
   }
-
+  const account_sid = credentials.account_sid;
```
Comment (davehorton) [inline lib/synth-audio.js:98]: oh hmm.  I did not think of this.  I don't think account_sid will be part of credentials, will it ?

Comment (xquanluu) [reply lib/synth-audio.js:98]: you are right. in https://github.com/jambonz/jambonz-feature-server/blob/main/lib/session/call-session.js#L585

the account_sid is not there, I will introduce new parameter to the function

Comment (xquanluu) [reply lib/synth-audio.js]: fixed

**Review by davehorton — CHANGES_REQUESTED** [review]
can you add test cases including where account sid is, and is not, provided?

**Comment by xquanluu** [comment]
In current testcases:

legacy testcase is for account_sid is null
Newly added testcase is for account_sid != null

So I dont think we need to add more testcase

**Review by davehorton — APPROVED** [review]

**Pull request merged**